spec/requests/get_services_spec.rb

```diff
@@ -23,24 +22,4 @@
     }
     expect(JSON.parse(response.body)).to include(JSON.parse(expected.to_json))
   end
-
-  it 'returns correct service data for services' do
```

Comment:
Removed these as they don't seem to reflect current functionality.  Will try to replace them once I understand how the ServicesAtLocation functionality works